index.html

```diff
@@ -16,6 +16,31 @@ <h1>Jelena de la Sauce</h1>
       <a href="#biography">Biography</a>
     </nav>
   </header>
+
+  <figure>
+    <img src="https://source.unsplash.com/random/400x200" alt=""> 
+    <figcaption>I am a well organized and communicative Web Development student, who is graduating the boothcamp at neuefische end of March 2022. I am higly motivated to join a young, creative and inspiring team.</figcaption>
+  </figure>
+
+  <section id="skills">
+    <p>  
```

Comment:
Good code but I think you shoudnt nest the h2 and ul element in p(remove the <p>). I think that the p elements can only contain phrasing content (for example elements like span and strong).